docs/book/driver-deployment/installation.md

```diff
@@ -181,14 +183,27 @@ rm csi-vsphere.conf
 
 Before you deploy the vSphere CSI driver, refer to the [Compatibility](../compatiblity_matrix.md) page to view the supported kubernetes versions for a particular vSphere CSI version and [feature support](../supported_features_matrix.md) page to see what features are supported on that version.
 
-- Choose the manifests pertaining to the latest version of vSphere CSI driver from the [Github](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/) repository. For example, if you want to deploy vSphere CSI v2.1.1 on a vSphere 7.0u1 environment, you will choose [this](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/v2.1.1/vsphere-7.0u1/vanilla) folder.
-  
+- Get the deployment manifests pertaining to the version of vSphere CSI driver.
+  - [v2.2.0](../releases/v2.2.0.md) deployment manifests - https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/v2.2.0/manifests/v2.2.0
+  - [v2.1.1](../releases/v2.1.1.md) deployment manifests - https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/v2.1.0
+  - [v2.1.0](../releases/v2.1.0.md) deployment manifests - https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/v2.1.0
```

Comment:
Should be using the master branch manifests or the released ones?

Reply:
used manifests from a tagged release for v2.2.0 and v2.1.1
https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/v2.2.0/manifests/v2.2.0
https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/v2.1.1/manifests/v2.1.1

for v2.1.0 could not find v2.1.0 manifests in the tag - https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/v2.1.0/manifests/